NewHaven/src/Resources.h

```diff
@@ -46,15 +46,17 @@ struct HarvestDeck
 struct Building
 {
 private:
-	const ResourceTypes* buildingType;	//Determines resource
-	const std::uint_least8_t* buildingNumber;
+	ResourceTypes* buildingType;	//Determines resource
+	std::uint_least8_t* buildingNumber;
 	bool* faceUp;	//Either the card is flipped or not
 
 public:
 	Building(ResourceTypes = ResourceTypes::SHEEP, std::uint_least8_t = 1, std::uint_least8_t = 0);
 	Building(const Building &building);
 	~Building();
 
+	Building& operator=(const Building&);
```

Comment:
`Building& operator=(const Building&)`

your operator is missing its parameter reference name. 
Suggested fix: 

`Building& operator=(const Building& building)`

Reply:
Yea, if you look closely I don't usually name the parameters in the header files.

Reply:
yeah it looked into, it threw me off at first but it seems that cpp doesnt really care whether its named or not in the header.